nest_elephant_tvb/translation/nest_to_tvb.py

```diff
@@ -84,6 +84,10 @@ def create_logger(path,name, log_level):
     ############# NEW Code end
     
     logger_master.info('clean file')
-    os.remove(path_to_files_receive)
-    os.remove(path_to_files_send)
+    # TODO: ugly solution, all MPI ranks want to delete, only the first one can.
```

Comment:
only the rank who create it should remove it.
This file are link to receive and send data. The MPI process link to it can be responsible for it.

Reply:
Marked as a TODO. will be resolved later